### PR TITLE
♻️ Refactored code to use encryption algorithm name from settings for consistency

### PR DESCRIPTION
**Pull request by sameeramin**

### CODE REVIEW

**Review by menkotoglou — APPROVED** [review]
I like it!

**Comment by sameeramin** [comment]
Thank you @menkotoglou, please let me know what else I need to do in order to get this PR merged.

**Comment by menkotoglou** [comment]
@sameeramin unfortunately I cannot do anything about it, since I'm not a repo maintainer.

File: backend/app/utils.py [hunk 1]
```diff
@@ -103,14 +103,16 @@ def generate_password_reset_token(email: str) -> str:
     encoded_jwt = jwt.encode(
         {"exp": exp, "nbf": now, "sub": email},
         settings.SECRET_KEY,
-        algorithm="HS256",
+        algorithm=settings.ALGORITHM,
```
Comment (tiangolo) [inline backend/app/utils.py:106]: This could be from `security.ALGORITHM`

File: backend/app/utils.py [hunk 2]
```diff
@@ -103,14 +103,16 @@ def generate_password_reset_token(email: str) -> str:
...
     )
     return encoded_jwt
 
 
 def verify_password_reset_token(token: str) -> str | None:
     try:
-        decoded_token = jwt.decode(token, settings.SECRET_KEY, algorithms=["HS256"])
+        decoded_token = jwt.decode(
+            token, settings.SECRET_KEY, algorithms=[settings.ALGORITHM]
```
Comment (tiangolo) [inline backend/app/utils.py:114]: This could be from `security.ALGORITHM`

**Review by tiangolo — CHANGES_REQUESTED** [review]
Thank you for the interest!

I don't think the algorithm should be configurable in the settings, it shouldn't really be variable. Let's better keep it in `security.py`.

But I agree that the backend utils should use the same variable from `security.py`. :nerd_face: Could you please update that?

**Comment by sameeramin** [comment]
> Thank you for the interest!
> 
> I don't think the algorithm should be configurable in the settings, it shouldn't really be variable. Let's better keep it in `security.py`.
> 
> But I agree that the backend utils should use the same variable from `security.py`. 🤓 Could you please update that?

Sure, I'll update that!

**Comment by sameeramin** [comment]
Hello @tiangolo, I have incorporated your suggestions

**Review by julioriffel — APPROVED** [review]

**Review by tiangolo — APPROVED** [review]

**Comment by tiangolo** [comment]
Great, thanks @sameeramin! :rocket: :cake:

**Pull request merged**